libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -23,6 +23,7 @@
 #include <AP_HAL/Util.h>
 #include <DataFlash/DataFlash.h>
 #include <GCS_MAVLink/GCS.h>
+#include <stdio.h>
```

Comment:
we used to avoid adding stdio.h as it broke on AVR, but now it is fine on all platforms, and it makes adding debug code easier